lib/dry/system/container.rb

```diff
@@ -169,6 +168,9 @@ def configured?
           @__configured__.equal?(true)
         end
 
+        # @api private
+        alias_method :import_direct_namespace, :import
```

Comment:
Our previous `.import` implementation still allowed  the original `Dry::Container.import` to be run by checking whether its single positional argument was a `Dry::Container::Namespace`. That's not possible with the new `.import` params signature, so I've preserved the original dry-container provided `.import` method via this alias.

TBH, though, I'm not sure whether it's worth keeping around at all. Thoughts?

Reply:
@timriley I don't think it is. I'd say we should reduce the codebase prior 1.0.0 final, otherwise such leftovers will haunt us later on.

Reply:
@solnic Cool, I might get rid of it even from this PR then. Thanks!